Directory.Build.targets

```diff
@@ -15,4 +15,13 @@
     <AnalysisMode>Recommended</AnalysisMode>
     <PackageReadmeFile>README.md</PackageReadmeFile>
   </PropertyGroup>
+
+  <PropertyGroup Condition="'$(SignAssembly)' == 'true'">
+    <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)graphql.snk</AssemblyOriginatorKeyFile>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(IsTestProject)' != 'true'">
```

Comment:
I would move this ItemGroup to another one to keep them together.

Reply:
The rest are PropertyGroup and so cannot be merged.